MastodonSDK/Sources/MastodonCore/FetchedResultsController/FollowedTagsFetchedResultController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  FollowedTagsFetchedResultController.swift
+//  
+//
+//  Created by Marcus Kida on 23.11.22.
+//
+
+import os.log
+import UIKit
+import Combine
+import CoreData
+import CoreDataStack
+import MastodonSDK
+
+public final class FollowedTagsFetchedResultController: NSObject {
+
+    var disposeBag = Set<AnyCancellable>()
+
+    let fetchedResultsController: NSFetchedResultsController<Tag>
+
+    // input
+    @Published public var domain: String? = nil
+    @Published public var user: MastodonUser? = nil
+
+    // output
+    let _objectIDs = CurrentValueSubject<[NSManagedObjectID], Never>([])
+    @Published public private(set) var records: [ManagedObjectRecord<Tag>] = []
+    
+    public init(managedObjectContext: NSManagedObjectContext, domain: String, user: MastodonUser) {
+        self.domain = domain
+        self.fetchedResultsController = {
+            let fetchRequest = Tag.sortedFetchRequest
+            fetchRequest.predicate = Tag.predicate(domain: domain, following: true, by: user)
+            fetchRequest.sortDescriptors = Tag.defaultSortDescriptors
+            fetchRequest.returnsObjectsAsFaults = false
+            fetchRequest.fetchBatchSize = 20
+            let controller = NSFetchedResultsController(
+                fetchRequest: fetchRequest,
+                managedObjectContext: managedObjectContext,
+                sectionNameKeyPath: nil,
+                cacheName: nil
+            )
+            
+            return controller
+        }()
+        super.init()
+        
+        // debounce output to prevent UI update issues
```

Comment:
This feels pretty hacky 😄

Reply:
This was copy & pasted but I'll remove it now and see how it behaves.